bounce.py

```diff
@@ -390,6 +394,19 @@ def _button_release_cb(self, win, event):
                     self._start_step()
         return True
 
+    def _mouse_motion_cb(self, widget, event):
+        if self.ball.grabbed and self._fraction is not None:
+            if event.is_hint:
+                mouse_x = event.window.get_pointer()[0]
```

Comment:
Calling `get_pointer` incurs a round-trip communication delay with the graphics system; is it really necessary to ascertain the actual position or is it sufficient to animate against the hinted position?